app/helpers/workflows_helper.rb

```diff
@@ -2,14 +2,6 @@
 
 # A few helpers used in pipeline workflows
 module WorkflowsHelper
-  # Returns descriptor from params, if it's not there try the @study.
```

Comment:
There was a lot of this spread all over the place. Haven't managed to remove it all, but have tried to consolidate a lot.